module/constants/activeEffectsTypes.js

```diff
@@ -120,25 +93,6 @@ export default {
       // }
     }
   },
```

Comment:
I think the one below we will have to keep for now, it is not specificaly  related to auras.
A character can become aligned to a realm by a virtue or flaw. If it is a magus, it will impact its magic.
Update: after rereading it the faery magic virtue, it is more complex than that. Magi of Merinita gain benefit of both fae and magic aura.
Let's scrap it, it needs some redesign, it was originaly there to indicate which realm a supernatural ability is impacted from, but now it is set at the ability level.